### PR TITLE
add documentation for capture body functionality

### LINKED ISSUE #452: Document CAPTURE_BODY and CAPTURE_BODY_CONTENT_TYPES configuration options
The feature was implemented in https://github.com/elastic/apm-agent-dotnet/pull/402 but we also need to document it. 
[Here is the link to the corresponding section of Java agent docs](https://www.elastic.co/guide/en/apm/agent/java/current/config-http.html#config-capture-body).

### PR DESCRIPTION
**Pull request by vhatsura**
closes #452 

@bmorelli25, can you review it, please?

### CODE REVIEW

**Comment by bmorelli25** [comment]
@elasticmachine run docs build

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -415,6 +415,57 @@ The default unit for this option is `s`.
...
+|============
+
+[options="header"]
+|============
+| Default                 | Type
+| `off`                  | String
+|============
+
+[float]
+[[config-capture-body]]
```
Comment (gregkalapos) [inline docs/configuration.asciidoc]: I think here is the problem - this should be `config-capture-body-content-types`.

Comment (bmorelli25) [reply docs/configuration.asciidoc]: Yup!

Comment (vhatsura) [reply docs/configuration.asciidoc]: fixed

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by gregkalapos** [comment]
Awesome, thanks everyone!

**Pull request merged**